### PR TITLE
CV-248569 PHP8 support

### PR DESCRIPTION
**Pull request by umpirsky**
Part of https://chesscom.atlassian.net/browse/CV-248569.

### Plan

- [x] QA it with main app
- [x] Merge

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,6 +31,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.0"
+        "phpunit/phpunit": "^8.0"
```
Comment (nikolaposa) [inline composer.json:34]: Why not `^9.5`?

**Review by nikolaposa — APPROVED** [review]
👍

File: composer.json (hunk 1 above)
Comment (umpirsky) [reply composer.json:34]: Because we have more important PHP things to do.

Comment (nikolaposa) [reply composer.json:34]: It's just a matter of version, everything else remains the same. Also, you've just switched from 5 to 8, no major BC breaks between PHPUnit 8 and 9. :)

Comment (nikolaposa) [reply composer.json:34]: > more important

I think **everything** is important, including PHPUnit. PHPUnit >= 9 has a better support for PHP 8.1 that we gonna migrate to soon.

Comment (umpirsky) [reply composer.json]: You are right, fixed in 03ab4eddd66c7623b9b9620ff82d4e4b7fa0972c

**Review by ikonic89 — APPROVED** [review]

**Pull request merged**